docs/wiki/[Variables]-configure_management_resources.md

```diff
@@ -1 +1,355 @@
+
+- Security
+- SecurityInsights
+
+See the [Azure Sentinel overview page][sentinel_overview] for more information.
+
+##### `settings.log_analytics.enable_solution_for_agent_health_assessment`
+
+Deploys the following `azurerm_log_analytics_solution`:
+
+- [AgentHealthAssessment][agent_health_overview]
+
+##### `settings.log_analytics.enable_solution_for_anti_malware`
+
+Deploys the following `azurerm_log_analytics_solution`:
+
+- AntiMalware
+
+##### `settings.log_analytics.enable_solution_for_azure_activity`
+
+Deploys the following `azurerm_log_analytics_solution`:
+
+- AzureActivity
+
+##### `settings.log_analytics.enable_solution_for_change_tracking`
+
+Deploys the following `azurerm_log_analytics_solution`:
+
+- [ChangeTracking][change_tracking_overview]
+
+##### `settings.log_analytics.enable_solution_for_service_map`
+
+Deploys the following `azurerm_log_analytics_solution`:
+
+- [ServiceMap][service_map_overview]
+
+##### `settings.log_analytics.enable_solution_for_sql_assessment`
+
+Deploys the following `azurerm_log_analytics_solution`:
+
+- [SQLAssessment][sql_assessment_overview]
+
+##### `settings.log_analytics.enable_solution_for_updates`
+
+Deploys the following `azurerm_log_analytics_solution`:
+
+- [Updates][updates_overview]
+
+##### `settings.log_analytics.enable_solution_for_vm_insights`
+
+Deploys the following `azurerm_log_analytics_solution`:
+
+- [VMInsights][vm_insights_overview]
+
+### Configure Microsoft Defender for Cloud
+
+Deploy Microsoft Defender for Cloud pricing tiers (previously Azure Defender and Azure Security Center) through Azure Policy assignment.
+
+```hcl
+security_center = {
+  enabled = true
+  config = {
+    email_security_contact             = "security_contact@replace_me"
+    enable_defender_for_acr            = true
+    enable_defender_for_app_services   = true
+    enable_defender_for_arm            = true
+    enable_defender_for_dns            = true
+    enable_defender_for_key_vault      = true
+    enable_defender_for_kubernetes     = true
+    enable_defender_for_oss_databases  = true
+    enable_defender_for_servers        = true
+    enable_defender_for_sql_server_vms = true
+    enable_defender_for_sql_servers    = true
+    enable_defender_for_storage        = true
+  }
+}
+```
+
+#### `settings.security_center.enabled`
+
+Enables or disables the `EnforcementMode` of the `Deploy-ASCDF-Config` policy assignment at the organizational root management group.
+
+#### `settings.security_center.config`
+
+The `config` (`object`) input allows you to set the following configuration items:
+
+##### `settings.security_center.email_security_contact`
+
+Specifies the email address to be used as the security contact in Microsoft Defender for Cloud.
+
+##### `settings.security_center.enable_defender_for_acr`
+
+Enables the the Standard pricing tier for `ContainerRegistry`.
+This is deployed to all in-scope subscriptions using the `DeployIfNotExists` policy effect.
```

Comment:
Is it worth calling out which policy/policies each of these settings effects?

Reply:
Yep - will add the policy definitions 👍